src/api/routes/offer.js

```diff
@@ -30,12 +32,16 @@ module.exports = (app) => {
     router.post("/", authMiddleware.isGod, validators.create, async (req, res) => {
         try {
             // This is safe since the service is destructuring the passed object and the fields have been validated
-            const offer = await (new OfferService()).create(req.body);
+            if (await Offer.find().activeOffersCount(req.body).length < Offer.MAX_ACTIVE_OFFERS_ALLOWED) {
+                const offer = await (new OfferService()).create(req.body);
+
+                return res.json(offer);
+            } else throw new Error(ValidationReasons.OFFER_LIMIT_REACHED(req.body.owner));
```

Comment:
By removing this if statement when creating the custom validator, do not forget to use this ValidatorReason as well